app/src/main/java/com/example/benchproject/app/movie/details/entity/MovieDetails.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.benchproject.app.movie.details.entity
+
+data class MovieDetails(
+    val id: Int,
+    val imageBackgroundUrl: String,
+    val name: String,
+    val genres: List<Genres>,
+    val description: String,
+    val budget: Int,
+    val rating: Double,
+    val releaseDate: String,
```

Comment:
You can use some of Date classes like `LocalDate`, `ZonedDateTime`, `LocalDateTime` depends on your needs

Reply:
At this moment this `relaseDate` looks like `2021-12-15` and if I use `LocalDate` for example I change it only in the module APP and this is not a bad idea? In `MovieDetailsRemote` and `MovieDetailsModel`, I used the type of String and should map `String` to `LocalDate` in module Data where I take data from API or I can map this when I do this in `Domain` -> `App`? 

If I parse `String` to `LocalData` (only for `APP` module) I will do it in `MovieDetailsModule` in function `toUi` and it will be looks like: 
```
@RequiresApi(Build.VERSION_CODES.O)
    fun toUi() = MovieDetails(
        id,
        imageBackgroundUrl,
        name,
        genres.toUi(),
        description,
        budget,
        rating,
        LocalDate.parse(releaseDate),
        imageUrl
    )
```
I must use this annotation because min require API is 26 and I have 23
![image](https://user-images.githubusercontent.com/57491794/177294759-834ad434-1298-433c-984a-ee27f17f34f0.png)

Reply:
And also I should use this annotation in every each places which I used this function

Reply:
Such mapping should be on at least in the domain layer. Depending on your needs in `data` you can keep it as a simple `String`.
Even if you would add this annotation then it wouldn't work on Android devices below Android O.
To tackle this you can use desugaring `https://developer.android.com/studio/write/java8-support`. 
You can use only `String` in this PR to not block rest of them and change `String` to `LocalDate` in a separate PR as a improvement.

Reply:
Ok, I understand and I created new task in issue #38 
![image](https://user-images.githubusercontent.com/57491794/177329896-dfddd389-6136-483c-a9d3-8713aca4ce4a.png)

Reply:
And now I back to `String` type in `relaseDate`